### ISSUE TITLE
Receive "Error updating plugins." every time I open Hyper

### ISSUE THREAD

**Issue by ryan4888** [zeit/hyper#1563]
I receive the following error every time I open Hyper. This has happened since install. 

![image](https://cloud.githubusercontent.com/assets/10523324/23141643/c9e0a86a-f785-11e6-9ab6-2f514bdcc286.png)

macOS 10.12
Hyper v1.2.1

**Comment by greggb** [comment]
https://github.com/zeit/hyper/issues/1513

**Comment by indexofrefraction** [comment]
and its being tracked here, too : https://github.com/zeit/hyper/issues/312
this is a mayor bug, I give up on hyper until this is solved.

**Comment by henrikdahl** [comment]
@indexofrefraction #312 is not the same as #1513

The commit that introduced the bug is only 1 months old and 5 months after #312 https://github.com/zeit/hyper/commit/993b8f6833bffe75032e9f161fd17bed0c98ed73

I fixed the issue 6 days ago https://github.com/zeit/hyper/pull/1516 and it's been merged. All we're waiting for is a release ✌️

**Comment by gsklee** [comment]
@henrikdahl Regarding #312 - is there any clue about the source of that bug?

I just did some tests on my machine (macOS):

* Not setting `shell`, no `nvm`-installed `node`, no system `node` : "Error updating plugins"
* Not setting `shell`, with `nvm`-installed `node`, no system `node` : "Error updating plugins"
* Setting `shell`, no `nvm`-installed `node`, no system `node` : "Cannot find `npm`"
* Setting `shell`, with `nvm`-installed `node`, no system `node` : Plugins updated successfully

So... at least on macOS, with the latest version of Hyper, the lack of a system `node` will not lead to the original error reported in #312. Maybe it's a Windows-specific issue?

**Comment by CodeTheory** [comment]
Closing as a duplicate of #1513. This will be fixed in an upcoming release of Hyper.
